pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+# What to add
+
+In order to approve this PR, we need the **title**, a **link** and a **category**.
+
+The current categories are:
```

Comment:
As I have mentioned in your Contributing PR, I think users would need to have a branch open to open a PR. So possibly they should update the readme themselves and open a PR for that.

If that is the flow, this template PR should be with checkboxes

- [ ] Title of the podcast is added to the proper category
- [ ] Link to the podcast is added to the entry